metacluster/include/metacluster/CreateTenant.actor.h

```diff
@@ -146,6 +149,9 @@ struct CreateTenantImpl {
 					throw invalid_tenant_configuration();
 				}
 				return std::make_pair(groupEntry.get().assignedCluster, true);
+			} else if (self->assignClusterAutomatically && self->ignoreCapacityLimit) {
```

Comment:
I would check for this as part of the `run` function, since these two parameters should be mutually exclusive always.

Reply:
What about the following case:
cluster A has a capacity of 1, and there is already a tenant group TG1.
Should we fail the following?
```
tenant create test_tenant tenant_group=TG1 ignore_capacity_limit?
```
Even without `assigned_cluster`, it still implicitly knows which cluster to assign the tenant to.

Reply:
See https://github.com/apple/foundationdb/pull/10173#discussion_r1188195600. Will update as suggested.

Reply:
Done